rules/windows/process_creation/win_susp_netsh_dll_persistence.yml

```diff
@@ -7,6 +7,7 @@ references:
 tags:
     - attack.persistence
     - attack.t1060
```

Comment:
Nice catch!

I think you could also remove the wrong one (run keys) tag and replace the reference to the correct one of your new tag: https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1128/T1128.md

Reply:
Sure thing